src/main/java/ContinuousIntegrationServer.java

```diff
@@ -141,55 +146,109 @@ public static void handle2(HttpExchange exchange)
         output.close();
     }
 
+    /**
+     * Fetches all builds from database and returns html template containing build url:s
+     * @param exchange
+     * @throws IOException
+     */
+    public static void buildHistory(HttpExchange exchange)
+            throws IOException
+    {
+        Headers headers = exchange.getResponseHeaders();
+        headers.set("ContentType", "text/html;charset=utf-8");
 
-    public void codeFetch(URL url){
-//        InputStream is = url.openStream();
+        BuildHistory buildHistory = new BuildHistory();
+        String response = "<html><body style=\"background:#FAFAFA;\"><h2>Build History: </h2><ul>";
 
-        try{
-//            System.out.print("hej");
-            BufferedReader reader = new BufferedReader(new InputStreamReader(url.openStream()));
-            String input;
-            while((input = reader.readLine()) != null){
-//                System.out.print("hej");
-                System.out.print(input);
-            }
-            reader.close();
-        }catch(Exception e){
+        ArrayList<BuildHistoryResult> builds = buildHistory.getAllBuilds();
+
+        String url = "https://ci-servergroup5.herokuapp.com/build";
```

Comment:
Change it to "/build" and it'll work both in production and locally.

Reply:
Smart! Fixed it now